### DISCUSSION TITLE
Diapositivas clase 20/9

### DISCUSSION THREAD

**Discussion by AgostinaApraiz** [INGCOM-UNRN-P1/cursada-2023#174] [Preguntas y respuestas] [answered]
Hola profes, no estaria encontrando las diapositivas del 20/9, estan subidas en alguna parte del campus?

Muchas gracias.
Agostina

**Comment by AnGys09** [comment, marked as answer]
Hola ahí ví que la subieron 😉

**Reply by martinvilu** [reply]
Vi la pregunta y las cargué, pero no hice tiempo de responder acá 

![dato-de-vital-importancia-te-lo-resumo](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/74bc27b1-e0c2-46d3-8aa7-4fc7a413129b)

[2023 - #8 - Programación 1.pdf](https://github.com/INGCOM-UNRN-P1/cursada-2023/files/12705446/2023.-.8.-.Programacion.1.pdf)
